spec/ruby/language/numbers_spec.rb

```diff
@@ -56,13 +56,13 @@
   end
 
   it "can be an octal literal with trailing 'r' to represent a Rational"  do
-    042.should == Rational(34, 1)
-    -042.should == Rational(-34, 1)
+    042r.should == Rational(34, 1)
+    -042r.should == Rational(-34, 1)
   end
 
   it "can be a binary literal with trailing 'r' to represent a Rational" do
-    0b1111.should == Rational(15, 1)
-    -0b1111.should == Rational(-15, 1)
+    0b1111r.should == Rational(15, 1)
+    -0b1111r.should == Rational(-15, 1)
```

Comment:
No, this spec was apparently broken, and it happens to just pass anyway. I'd push it to RubySpec if it still existed!